### ISSUE TITLE
-k doesn't work

### ISSUE THREAD

**Issue by sscarduzio** [libetl/curl#4]
Hello, thanks for this library, really handy.

I have this error:

`... Caused by: org.toilelibre.libe.curl.Curl$CurlException: java.util.concurrent.ExecutionException: org.toilelibre.libe.curl.Curl$CurlException: javax.net.ssl.SSLPeerUnverifiedException: Host name 'localhost' does not match the certificate subject provided by the peer (CN=Unknown, OU=Unknown, O=Unknown, L=Unknown, ST=Unknown, C=Unknown)`

when I do:

```scala
curl.k.xUpperCase("GET").run(url)
```
I'm calling this from Scala, that's why there's no parens. 

I have the same result with:

```scala
$(s"curl -k -u admin:container $url")
```

**Comment by libetl** [comment]
Hi,
Intriguing, let me check.

**Comment by libetl** [comment]
You are right, testing

```bash
curl -k https://wrong.host.badssl.com/
```
fails

**Comment by libetl** [comment]
Ok I know what is wrong.

**Comment by libetl** [comment]
released : v0.0.13
Badge on the README.md to be refreshed in a couple of hours.

**Comment by sscarduzio** [comment]
that was fast, thanks @libetl!
